manifest-parser.js

```diff
@@ -229,7 +229,11 @@ var ManifestParser = (function() {
     try {
       _json_input = JSON.parse(string);
     } catch (e) {
-      _logs.push("File isn't valid JSON: " + e);
+      if (e.message === "Unexpected token '") {
+        _logs.push("File isn't valid JSON: Double quotes are considered valid JSON, single quotes aren't. Please use double quotes.");
```

Comment:
Should we try to find the line + character number from the original error message? Or maybe show it for information? I'm afraid the devs are losing interesting info.